tests/test_adapter.py

```
@@ -339,6 +339,109 @@ def test_api_alter_table_change_primary_key_name(engine, capsys):
+    Invoke gRPC `Migrate` without operation.
+    """
+
+    # Invoke gRPC API method under test.
+    response = migration_request()
+
+    # Validate outcome.
+    assert response.success is False
+    assert response.unsupported is True
+
+    # Validate log output.
+    out, err = capsys.readouterr()
```

Ruff (0.14.13)

Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)